trappy/base.py

```diff
@@ -179,6 +179,12 @@ def append_data(self, time, comm, pid, cpu, line, data):
         self.line_array.append(line)
         self.data_array.append(data)
 
+    def conv_to_int(self, value):
+        # Handle false-positives for negative numbers
+        if value.lstrip("-").isdigit():
+            value = int(value)
```

Comment:
Can you give an example? I would rather it failed if `isdigit` returned true and it still failed because then we can find those cases and make it more robust. Also *I think* using try/except will slow it down so I'd rather not do it in the hot path...